CHANGELOG.md

```diff
@@ -67,6 +67,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   the jj repo from a Git repo. They will now be considered as non-existent if
   referenced explicitly instead of crashing.
 
+* Trailing spaces in `.gitignore` must be kept only if properly escaped.
```

Comment:
How about rephrasing to something like "Fixed handling of escaped characters in .gitignore (only keep trailing spaces if escaped properly)"?

The "must" is a little confusing to interpret in a Changelog entry.

Reply:
Much better indeed.